docs/user/resources/02-logpipeline.md

```diff
@@ -91,6 +91,7 @@ For details, see the [LogPipeline specification file](https://github.com/kyma-pr
 | **input.&#x200b;application.&#x200b;containers.&#x200b;include**  | \[\]string | Specifies to include only the container logs with the specified container names. |
 | **input.&#x200b;application.&#x200b;dropLabels**  | boolean | Defines whether to drop all Kubernetes labels. The default is `false`. |
 | **input.&#x200b;application.&#x200b;keepAnnotations**  | boolean | Defines whether to keep all Kubernetes annotations. The default is `false`. |
+| **input.&#x200b;application.&#x200b;keepRawBody**  | boolean | If the `log` attribute contains a JSON payload and it is successfully parsed, the `log` attribute will be retained if `keepRawBody` is set to `true`. Otherwise, the log attribute will be removed from the log record. Default is `true`. |
```

Comment:
```suggestion
| **input.&#x200b;application.&#x200b;keepRawBody**  | boolean | If the `log` attribute contains a JSON payload and it is successfully parsed, the `log` attribute is retained if `keepRawBody` is set to `true`. Otherwise, the log attribute is removed from the log record. Default is `true`. |
```